test/decompile/basic.txt

```diff
@@ -119,8 +119,8 @@ import table ns_tab3:funcref;
 table T_b:funcref(min: 0, max: 10);
 export table tab2:funcref(min: 0, max: 11);
 
-data d_a(offset: 0) = "Hello, World!\0a\00";
-data d_b(offset: 100) = 
+data d_HelloWorld(offset: 0) = "Hello, World!\0a\00";
+data d_abcdefghijklmnoqrstuvwxyzabc(offset: 100) = 
```

Comment:
Oh it wasn't obvious to me that the maximum would give a truncated identifier. I thought it would be symmetric with the minimum and not generate any name.

Reply:
commented